integration/testutil.go

```diff
@@ -290,14 +295,21 @@ func withPlatform(p spec.Platform) imageOpt {
 }
 
 type imageInfo struct {
+	imgName   string
 	ref       string
+	mirror    bool
 	creds     string
 	plainHTTP bool
 	platform  spec.Platform
 }
 
 func dockerhub(name string, opts ...imageOpt) imageInfo {
-	i := imageInfo{dockerLibrary + name, "", false, platforms.DefaultSpec()}
+	i := imageInfo{
+		imgName: name[:strings.IndexByte(name, ':')],
```

Comment:
this doesn't work if the image name contains a registry port (e.g. `localhost:5000/rabbitmq:latest` in our get start doc).

Reply:
A helper to break apart the image ref into components would be valuable. I also suspect containerd might already have one.

Reply:
Good point , although calls to `dockerhub` assume the image to be residing in ECR. Calls to `mirror()` would break though if our local registry happens to contain ':'

Reply:
Actually looking back, this shouldnt break at all since we are splitting the image name+tag IE (rabbitmq:1.x or rabbitmq@sha256) and not the ref IE (localhost:5000/rabbitmq:latest)